audio/midi_drivers/XMidiFile.cpp

```diff
@@ -1195,13 +1284,16 @@ int XMidiFile::CreateMT32SystemMessage(
 		const void* data, IDataSource* source) {
 	CreateNewEvent(time);
 	// SysEx status
-	current->status = 0xF0;
+	current->status = int(MidiStatus::Sysex);
```

Comment:
Would it make sense to make status be of the enum type instead?

Reply:
Not really, status doesn't just contain valid enum values it is (MidiStatus | channel_number)